src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1122,6 +1126,39 @@ const formConfig = {
             ),
           }),
         },
+        page18f1: {
+          path: 'applicant-information/:index/marriage-details',
+          arrayPath: 'applicants',
+          showPagePerItem: true,
+          title: item => `${applicantWording(item)} marriage documents`,
+          depends: (formData, index) => {
+            if (index === undefined) return true;
+            return (
+              ['spouse', 'spouseSeparated'].includes(
+                get(
+                  'applicantRelationshipToSponsor.relationshipToVeteran',
+                  formData?.applicants?.[index],
+                ),
+              ) && get('sponsorIsDeceased', formData)
+            );
+          },
+          CustomPage: ApplicantSponsorMarriageDetailsPage,
+          CustomPageReview: ApplicantSponsorMarriageDetailsReviewPage,
+          schema: applicantListSchema([], {
+            applicantSponsorMarriageDetails: {
+              type: 'object',
+              properties: {
+                relationshipToVeteran: { type: 'string' },
+                otherRelationshipToVeteran: { type: 'string' },
+              },
+            },
+          }),
+          uiSchema: {
+            applicants: {
+              items: {},
+            },
+          },
+        },
```

Comment:
Add new custom page that only displays when sponsor is deceased and applicant was married to sponsor at some point. This is where we will collect additional details about the marriage to determine if they need to give us a certificate.